ART - Supp App - Artist Community

GaboHub opened the pull request:
This include a refactor to the android resources to this sup app.

The artist Community is working correctly, but needs futher fixes
#5879

darkestpriest commented:
**Issues to resolve:**
* The app throws exceptions when I pushed back button.
`========================================================================================================================================================
01-31 21:44:52.174 1689-1689/com.bitdubai.fermat W/System.err: Fermat Error Manager * Unexpected Exception Report
01-31 21:44:52.174 1689-1689/com.bitdubai.fermat W/System.err: ========================================================================================================================================================
01-31 21:44:52.174 1689-1689/com.bitdubai.fermat W/System.err: Source: ACTIVITY
01-31 21:44:52.174 1689-1689/com.bitdubai.fermat W/System.err: Severity: UNSTABLE
01-31 21:44:52.174 1689-1689/com.bitdubai.fermat W/System.err: ********************************************************************************************************************************************************
01-31 21:44:52.174 1689-1689/com.bitdubai.fermat W/System.err: Exception Number: 1
01-31 21:44:52.174 1689-1689/com.bitdubai.fermat W/System.err: Exception Type: class java.lang.IllegalArgumentException
01-31 21:44:52.174 1689-1689/com.bitdubai.fermat W/System.err: Exception Message: Error in changeActivity
01-31 21:44:52.175 1689-1689/com.bitdubai.fermat W/System.err: Exception Possible Cause: N/A
01-31 21:44:52.175 1689-1689/com.bitdubai.fermat W/System.err: Exception Context: 
01-31 21:44:52.175 1689-1689/com.bitdubai.fermat W/System.err: ---------------------------------------------------------------------------------
01-31 21:44:52.175 1689-1689/com.bitdubai.fermat W/System.err: N/A
01-31 21:44:52.175 1689-1689/com.bitdubai.fermat W/System.err: ---------------------------------------------------------------------------------
01-31 21:44:52.175 1689-1689/com.bitdubai.fermat W/System.err: Exception Stack Trace: 
01-31 21:44:52.175 1689-1689/com.bitdubai.fermat W/System.err: ---------------------------------------------------------------------------------
01-31 21:44:52.175 1689-1689/com.bitdubai.fermat W/System.err: Class: com.bitdubai.android_core.app.SubAppActivity - Line: 361
01-31 21:44:52.175 1689-1689/com.bitdubai.fermat W/System.err: Class: com.bitdubai.android_core.app.SubAppActivity - Line: 227
01-31 21:44:52.175 1689-1689/com.bitdubai.fermat W/System.err: Class: android.app.Activity - Line: 2456
01-31 21:44:52.175 1689-1689/com.bitdubai.fermat W/System.err: Class: android.view.KeyEvent - Line: 2643
01-31 21:44:52.175 1689-1689/com.bitdubai.fermat W/System.err: Class: android.app.Activity - Line: 2707
01-31 21:44:52.175 1689-1689/com.bitdubai.fermat W/System.err: Class: android.support.v7.view.WindowCallbackWrapper - Line: 50
01-31 21:44:52.175 1689-1689/com.bitdubai.fermat W/System.err: Class: android.support.v7.app.AppCompatDelegateImplBase$AppCompatWindowCallbackBase - Line: 224
01-31 21:44:52.175 1689-1689/com.bitdubai.fermat W/System.err: Class: android.support.v7.view.WindowCallbackWrapper - Line: 50
01-31 21:44:52.175 1689-1689/com.bitdubai.fermat W/System.err: Class: com.android.internal.policy.impl.PhoneWindow$DecorView - Line: 2276
01-31 21:44:52.175 1689-1689/com.bitdubai.fermat W/System.err: Class: android.view.ViewRootImpl$ViewPostImeInputStage - Line: 4020
01-31 21:44:52.175 1689-1689/com.bitdubai.fermat W/System.err: Class: android.view.ViewRootImpl$ViewPostImeInputStage - Line: 3982
01-31 21:44:52.175 1689-1689/com.bitdubai.fermat W/System.err: Class: android.view.ViewRootImpl$InputStage - Line: 3544
01-31 21:44:52.175 1689-1689/com.bitdubai.fermat W/System.err: Class: android.view.ViewRootImpl$InputStage - Line: 3597
01-31 21:44:52.175 1689-1689/com.bitdubai.fermat W/System.err: Class: android.view.ViewRootImpl$InputStage - Line: 3563
01-31 21:44:52.175 1689-1689/com.bitdubai.fermat W/System.err: Class: android.view.ViewRootImpl$AsyncInputStage - Line: 3680
01-31 21:44:52.175 1689-1689/com.bitdubai.fermat W/System.err: Class: android.view.ViewRootImpl$InputStage - Line: 3571
01-31 21:44:52.175 1689-1689/com.bitdubai.fermat W/System.err: Class: android.view.ViewRootImpl$AsyncInputStage - Line: 3737
01-31 21:44:52.175 1689-1689/com.bitdubai.fermat W/System.err: Class: android.view.ViewRootImpl$InputStage - Line: 3544
01-31 21:44:52.175 1689-1689/com.bitdubai.fermat W/System.err: Class: android.view.ViewRootImpl$InputStage - Line: 3597
01-31 21:44:52.175 1689-1689/com.bitdubai.fermat W/System.err: Class: android.view.ViewRootImpl$InputStage - Line: 3563
01-31 21:44:52.175 1689-1689/com.bitdubai.fermat W/System.err: Class: android.view.ViewRootImpl$InputStage - Line: 3571
01-31 21:44:52.175 1689-1689/com.bitdubai.fermat W/System.err: Class: android.view.ViewRootImpl$InputStage - Line: 3544
01-31 21:44:52.175 1689-1689/com.bitdubai.fermat W/System.err: Class: android.view.ViewRootImpl$InputStage - Line: 3597
01-31 21:44:52.175 1689-1689/com.bitdubai.fermat W/System.err: Class: android.view.ViewRootImpl$InputStage - Line: 3563
01-31 21:44:52.175 1689-1689/com.bitdubai.fermat W/System.err: Class: android.view.ViewRootImpl$AsyncInputStage - Line: 3713
01-31 21:44:52.175 1689-1689/com.bitdubai.fermat W/System.err: Class: android.view.ViewRootImpl$ImeInputStage - Line: 3874
01-31 21:44:52.175 1689-1689/com.bitdubai.fermat W/System.err: Class: android.view.inputmethod.InputMethodManager$PendingEvent - Line: 2208
01-31 21:44:52.175 1689-1689/com.bitdubai.fermat W/System.err: Class: android.view.inputmethod.InputMethodManager - Line: 1849
01-31 21:44:52.175 1689-1689/com.bitdubai.fermat W/System.err: Class: android.view.inputmethod.InputMethodManager - Line: 1840
01-31 21:44:52.175 1689-1689/com.bitdubai.fermat W/System.err: Class: android.view.inputmethod.InputMethodManager$ImeInputEventSender - Line: 2185
01-31 21:44:52.175 1689-1689/com.bitdubai.fermat W/System.err: Class: android.view.InputEventSender - Line: 141
01-31 21:44:52.175 1689-1689/com.bitdubai.fermat W/System.err: Class: android.os.MessageQueue - Line: -2
01-31 21:44:52.175 1689-1689/com.bitdubai.fermat W/System.err: Class: android.os.MessageQueue - Line: 143
01-31 21:44:52.175 1689-1689/com.bitdubai.fermat W/System.err: Class: android.os.Looper - Line: 122
01-31 21:44:52.175 1689-1689/com.bitdubai.fermat W/System.err: Class: android.app.ActivityThread - Line: 5254
01-31 21:44:52.175 1689-1689/com.bitdubai.fermat W/System.err: Class: java.lang.reflect.Method - Line: -2
01-31 21:44:52.175 1689-1689/com.bitdubai.fermat W/System.err: Class: java.lang.reflect.Method - Line: 372
01-31 21:44:52.175 1689-1689/com.bitdubai.fermat W/System.err: Class: com.android.internal.os.ZygoteInit$MethodAndArgsCaller - Line: 903
01-31 21:44:52.175 1689-1689/com.bitdubai.fermat W/System.err: Class: com.android.internal.os.ZygoteInit - Line: 698
01-31 21:44:52.175 1689-1689/com.bitdubai.fermat W/System.err: ---------------------------------------------------------------------------------
01-31 21:44:52.175 1689-1689/com.bitdubai.fermat W/System.err: Exceptions Processed: 1
01-31 21:44:52.175 1689-1689/com.bitdubai.fermat W/System.err: =================================================================================================================================================`
Please, check the navigation structure.
* Remove the hardcodes, we need to work with real Fermat actors.
* The error messages must be in English.
CC: @franklinmarcano1970 @lostwood

GaboHub committed:
REFACTOR: Include Artist Community XML files refactor.

This include the new fields and file names to this community to work.
​
Included in: #5879

Signed-off-by: Gabriel Araujo <gabe_512@hotmail.com>

GaboHub committed:
REFACTOR: Include Artist Communitu java classes.

This include the refactor the android R class to the resources.
​
Included in: #58795886

Signed-off-by: Gabriel Araujo <gabe_512@hotmail.com>
